### DISCUSSION TITLE
searx.org question

### DISCUSSION THREAD

**Discussion by lm2lm2** [searxng/searxng#4110] [Q&A]
hello

who is the owner of searx.org?

i see that it's the rare (or only) isntance to be never blocked by various search engines, especially google.

I really wonder who is behind them? where almost all searx instances indicates well the "[Contact instance maintainer](mailto:@)", there is nothing found on the searx.org one.
Is it possible to know much more who, or how to contact, or manages, each searx engine? like a mandatory criteria to make public searx instance?

thank you

**Comment by Bnyro** [comment]
You should ask in the community matrix room of SearXNG, searx.org has no affiliation with the developers of SearXNG.
